pkg/rpcclient/wsclient_test.go

```diff
@@ -574,6 +575,72 @@ func TestWSFilteredSubscriptions(t *testing.T) {
 				require.Equal(t, util.Uint256{1, 2, 3}, *filt.Container)
 			},
 		},
+		{
+			"notary request sender",
+			func(t *testing.T, wsc *WSClient) {
+				sender := util.Uint160{1, 2, 3, 4, 5}
+				_, err := wsc.ReceiveNotaryRequests(&neorpc.NotaryRequestFilter{Sender: &sender}, make(chan *result.NotaryRequestEvent))
+				require.NoError(t, err)
+			},
+			func(t *testing.T, p *params.Params) {
+				param := p.Value(1)
+				filt := new(neorpc.NotaryRequestFilter)
+				require.NoError(t, json.Unmarshal(param.RawMessage, filt))
+				require.Equal(t, util.Uint160{1, 2, 3, 4, 5}, *filt.Sender)
+				require.Nil(t, filt.Signer)
+			},
+		},
+		{
+			"notary request signer",
+			func(t *testing.T, wsc *WSClient) {
+				signer := util.Uint160{0, 42}
+				_, err := wsc.ReceiveNotaryRequests(&neorpc.NotaryRequestFilter{Signer: &signer}, make(chan *result.NotaryRequestEvent))
+				require.NoError(t, err)
+
+			},
+			func(t *testing.T, p *params.Params) {
+				param := p.Value(1)
+				filt := new(neorpc.NotaryRequestFilter)
+				require.NoError(t, json.Unmarshal(param.RawMessage, filt))
+				require.Nil(t, filt.Sender)
+				require.Equal(t, util.Uint160{0, 42}, *filt.Signer)
+			},
+		},
+		{
+			"notary request type",
+			func(t *testing.T, wsc *WSClient) {
+				mempoolType, err := mempoolevent.GetEventTypeFromString("added")
```

Comment:
No need to use this helper, just use `mempoolevent.TansactionAdded`, we're testing the way how user will use this API.